src/FileConverter.ts

```diff
@@ -380,4 +378,13 @@ export class FileConverter {
 
         return FileConverter.parseVersion(rawSchemaVersion);
     }
+
+    /**
+     * Returns the path to the multi-tool.
+     * The path can be overridden by setting 'sarifViewer.multiToolPath'
+     * in the process environment.
+     */
+    private static get pathToMulitTool(): string {
```

Comment:
sp

Reply:
Why `pathToMultiTool` instead of `multiToolPath`? There's no naming collision.

Reply:
This was due to the "multiToolPath" "default import" above.

I've aliased the default import to "nodeModulesMultiToolPath" and change this property name to "multiToolPath" to make it very clear what this doing.